src/containers/views/Configuration.js

```diff
@@ -73,6 +73,7 @@ export class Configuration extends Component {
     return (
       <HotKeys handlers={keyboardHandlers} className="single">
         {errors && errors.length > 0 && <Errors errors={errors} />}
+        {setTitleTag('Configuration')}
```

Comment:
Ah.. that.. well, the thing is for `*Edit` views, I had faced errors when trying to set the title with `this.props.[entity].title` probably because the state might have been still at `isFetching`.. so decided to add it to the `render()` after `isFetching` is set to `false`.. then followed same for all other views..